autosubmit/job/job_list_persistence.py

```diff
@@ -111,7 +111,7 @@ def save(self, persistence_path, persistence_file, job_list, graph):
         setrecursionlimit(500000000)
         Log.debug("Saving JobList: " + path)
         with open(path, 'wb') as fd:
-            pickle.dump(graph, fd, pickle.HIGHEST_PROTOCOL)
+            pickle.dump({job.name: job.getstate() for job in job_list}, fd, pickle.HIGHEST_PROTOCOL)
```

Comment:
:+1: , @LuiggiTenorioK , FYI the dict Dani mentioned yesterday in the meeting :point_up: